rero/rero-ils
# US965: Holdings/items for ebooks

- fixture: implement ebooks holdings rero-ils and unit test fixtures
- ebooks: create holdings automatically after record harvesting
- form options: item type and location

* Adds online locations.
* Adds online item types.
* Adds circulation policies for ebooks.
* Adds online type to item types resource.
* Adds is_online to location resource.
* Adds online_harvested_source to the organisation resource.
* Adds extended validation for the resource location field is_online.
* Adds extended validation for the resource item_type field type.
* Adds an option to setup script to continue process after holding_items generation.
* Fixes some variables snake cases.
* Fixes typos.
* Adds custom widgets
* Adds a new column "type" on item_type
* Adds a new column "is_online" on location

## How to test?

- `./script/bootstrap`
- `./script/setup` (with harvested ebook)
- Document detail view: check ebook link below holding
- Item Type Form: new option `online` (unicity)
- Location form: new option `is_online` (unicity)

Co-Authored-by: Aly Badr <aly.badr@rero.ch>
Co-Authored-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>
Co-Authored-by: Alicia Zangger <alicia.zangger@rero.ch>
